.github/ISSUE_TEMPLATE/patch_release.md

```diff
@@ -22,27 +22,31 @@ this issue for posterity. Assuming `vX.Y.Z-up.K` is being cut, after upstream
 [crossplane/crossplane][upstream-xp] `vX.Y.Z` has been released
 according to the declared schedule, you should have:
 
-- [ ] Synced the `release-vX.Y` release branch in [upbound/crossplane][upbound-xp-fork], with upstream [crossplane/crossplane][upstream-xp] release branch, up to the `vX.Y.Z` tag, adding any required change specific to the fork, see [here][sync-xp-fork] for more details.
+- [ ] Synced the `release-X.Y` release branch in [upbound/crossplane][upbound-xp-fork], with upstream [crossplane/crossplane][upstream-xp] release branch, up to the `vX.Y.Z` tag, adding any required change specific to the fork, see [here][sync-xp-fork] for more details.
 - [ ] Tagged [upbound/crossplane][upbound-xp-fork] `vX.Y.Z-up.K` from the `release-X.Y` branch by:
-  - [ ] Running the [Tag workflow][tag-xp-fork] on the `release-vX.Y` branch with the proper release version, `vX.Y.Z-up.K`. Use `Release vX.Y.Z-up.K` as message (FYI: the format suggested is only for consistency, there is no actual dependency on it).
-  - [ ] Running the [CI workflow][ci-xp-fork] on the `release-vX.Y` branch to build and publish the latest tagged artifacts.
+  - [ ] Running the [Tag workflow][tag-xp-fork] on the `release-X.Y` branch with the proper release version, `vX.Y.Z-up.K`. Use `Release vX.Y.Z-up.K` as message (FYI: the format suggested is only for consistency, there is no actual dependency on it).
+  - [ ] Running the [CI workflow][ci-xp-fork] on the `release-X.Y` branch to build and publish the latest tagged artifacts.
   - [ ] Confirm the image has been successfully published using: `docker buildx imagetools inspect docker.io/upbound/crossplane:vX.Y.Z-up.K`
```

Comment:
Would it be helpful to define variables for the releases when we are using commands? This could lead to less _copy-paste_ errors. Something like this. We could define those variables at the beginning. This specific version is used later in multiple commands.

```bash
export RELEASE_TAG=v1.15.1-up.1
docker buildx imagetools inspect docker.io/upbound/crossplane:"${RELEASE_TAG}"
```